searx/templates/simple/macros.html

```diff
@@ -42,7 +42,7 @@ <h3>{{ result_link(result.url, result.title|safe) }}</h3>
 {%- macro result_sub_footer(result, proxify) -%}
 <div class="engines">
   {% for engine in result.engines %}<span>{{ engine }}</span>{% endfor %}
-  {{ result_link(cache_url + result.url, icon_small('ellipsis-vertical-outline') + _('cached'), "cache_link") }}&lrm; {% if proxify and proxify_results %} {{ result_link(proxify(result.url), icon('link') + _('proxied'), "proxyfied_link") }} {% endif %}
+  {{ icon_small('ellipsis-vertical-outline') + result_link(cache_url + result.url, _('cached'), "cache_link") }}&lrm; {% if proxify and proxify_results %}&emsp;{{ result_link(proxify(result.url), icon('link') + _('proxied'), "proxyfied_link") }} {% endif %}
```

Comment:
The `&emsp;` is responsible for a (IMO) to great space between cache and proxy link

![image](https://github.com/searxng/searxng/assets/554536/79201957-0ba9-4f0d-8e9b-6c9b08193957)

I haven't look at the details, but may we get a better result, when we use a `<span>` tag around the links (like we use one line above to enumerate the engines)?

Reply:
Whoops, my bad. It was meant to be &ensp; thanks for catching this. Will fix.

Regarding wrapping the whole line in a container, I didn't have much success with this approach. Icon within <a> breaks vertical alignment with or without container for me, so I moved it outside.